### DISCUSSION TITLE
Playtimetop placeholders

### DISCUSSION THREAD

**Discussion by trickman325** [Zrips/CMI#2692] [Feature Requests]
**Placeholders for Player playtime top 1-10**

**%cmi_playtimetop_player_#%**
this would show which player is in the number slot you put (from 1-10)

**%cmi_playtimetop_time_#%**
This would show you the time played in seconds of the player in the slot selected (from 1-10)

**%cmi_playtimetop_time_formatted_#%**
This would show the time played formatted into days, hours, minutes, seconds of the slot selected (from 1-10)

These placeholders would just work off the first page of the top playtime list.

These would be really handy in making leaderboards of the most active players on the server.
@Zrips

**Comment by mercurialmusic** [comment]
Duplicates #645

**Comment by trickman325** [comment]
@Zrips 
You've added seconds playtime placeholders can you add these please?
